internal/rpc/user_devices.go

```diff
@@ -635,3 +637,21 @@ func (s *userDeviceRPCServer) UpdateUserDeviceMetadata(ctx context.Context, req
 
 	return &emptypb.Empty{}, nil
 }
+
+func (s *userDeviceRPCServer) ClearMetaTransactionRequests(ctx context.Context, _ *emptypb.Empty) (*pb.ClearMetaTransactionRequestsResponse, error) {
+	currTime := time.Now()
+	fifteenminsAgo := currTime.Add(-time.Minute * 15)
+
+	m, err := models.MetaTransactionRequests(
+		models.MetaTransactionRequestWhere.CreatedAt.LT(fifteenminsAgo),
```

Comment:
Since this is not an automated step, I don't think we need to guarantee that the request has been stuck for 15 minutes.

Reply:
[https://linear.app/dimo/issue/SI-2496/clear-stuck-transactions-automatically-after-15-minutes
](https://linear.app/dimo/issue/SI-2496/clear-stuck-transactions-automatically-after-15-minutes
)
The ticket Seems to suggest that, no other parameter were given on conditions to clear the transaction other than the length of time.

Reply:
I think it was quite clear though, that the time it has been on queue is the most important thing. But anyways, I believe this has been clarified.